Formula/gambit-scheme.rb

```diff
@@ -0,0 +1,28 @@
+class GambitScheme < Formula
+  desc "Gambit Scheme is an implementation of the Scheme Language"
+  homepage "http://gambitscheme.org"
+  url "https://github.com/gambit/gambit/archive/v4.9.1.tar.gz"
+  sha256 "667ae2ee657c22621a60b3eda6e242224d41853adb841e6ff9bc779f19921c18"
+
+  depends_on "openssl"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --enable-single-host
+      --enable-multiple-versions
+      --enable-default-runtime-options=f8,-8,t8
+      --enable-poll
+      --enable-openssl
+    ]
+
+    system "./configure", *args
+    system "make"
+    ENV.deparallelize
```

Comment:
Upstream is aware of the concurrent git issue found with using brew, but not triggered via `make -j8 install`

Reply:
Can you detail why the install process is using `git`? If it is downloading things, we generally do not allow that.

Reply:
Just reflecting the error only seen under brew, and not reproducible with `make -j8 install`